code/game/atoms.dm

```diff
@@ -1490,7 +1490,7 @@
 				created_atom.pixel_y += rand(-8,8)
 			SEND_SIGNAL(created_atom, COMSIG_ATOM_CREATEDBY_PROCESSING, src, chosen_option)
 			created_atom.OnCreatedFromProcessing(user, process_item, chosen_option, src)
-			to_chat(user, span_notice("You manage to create [chosen_option[TOOL_PROCESSING_AMOUNT]] [initial(atom_to_create.name)]\s from [src]."))
+			to_chat(user, span_notice("You manage to create [chosen_option[TOOL_PROCESSING_AMOUNT]] [initial(atom_to_create.gender) == PLURAL ? "[initial(atom_to_create.name)]" : "[initial(atom_to_create.name)]\s"] from [src]."))
```

Comment:
Sorry for not being more concise. 
This should be automatic with` \s`, I meant you need to set the gender of the onions.

Reply:
ahhh I thought you tried to imply that they are supposed to be plural

Reply:
Oh crap my bad. I could swear I checked & they didn't have the gender var set to plural. This was the right fix lmao

Reply:
i see alright

Reply:
done